HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -40,11 +40,19 @@ def customiseFor17792(process):
             producer.produceSeedStopReasons = cms.bool(False)
     return process
 
+def customiseFor18118(process):
+    hbhereconames = ['hltHbhePhase1Reco','hltHbherecoMethod2L1EGSeeded','hltHbherecoMethod2L1EGUnseeded']
+    for hbhereconame in hbhereconames:
```

Comment:
Please use a loop over producers_by_type using "HBHEPhase1Reconstructor" - this is safer than module labels which may change!